services/src/main/java/org/exoplatform/officeonline/AbstractOfficeOnlineService.java

```diff
@@ -143,6 +155,35 @@ public AbstractOfficeOnlineService(SessionProviderService sessionProviders,
     this.documentService = documentService;
     this.keyCache = cacheService.getCacheInstance(KEY_CACHE_NAME);
     this.userACL = userACL;
+    initFileExtensions();
+  }
+
+  /**
+   * Inits the file extensions.
+   */
+  protected void initFileExtensions() {
+    fileExtensions.put("application/msword", "doc");
```

Comment:
Does it have a sense to make this configurable? Can it vary, be extended in future or may be an user may want to limit supported exts?

Reply:
Added configurable DocumentTypePlugin.
Commit: https://github.com/exo-addons/office-online/pull/15/commits/0ea08afcf3872d8b6b41acb5a53931aaea00280b